CONTRIBUTING.md

```diff
@@ -142,22 +142,21 @@ docker run \
 	--env ZIMFARM_CPUS='3' \
 	--env ZIMFARM_MEMORY='2G' \
 	--env ZIMFARM_DISK='2G' \
-	--env SELFISH="y" \
+	--env SELFISH="true" \
 	--env USERNAME='admin' \
-	--env DEBUG=1 \
+	--env DEBUG=true \
```

Comment:
Add quote around `true` + use single quotes everywhere (mostly just for consistency, probably doesn't change anything)